python/cog/director/__main__.py

```diff
@@ -47,7 +47,7 @@ def _die(signum: Any, frame: Any) -> None:
 
 args = parser.parse_args()
 
-events = queue.Queue(maxsize=128)
+events: queue.Queue = queue.Queue(maxsize=128)
```

Comment:
lol, so glad that the python typing ecosystem has such powerful type inference :P

Reply:
Seems to have problems with `queue.Queue` but basically nothing else 🤷